### PR TITLE
Fix documentation

### PR DESCRIPTION
**Pull request by Enrique-C**

### CODE REVIEW

File: src/main/java/com/jalasoft/jfc/model/image/ImageConverter.java [hunk 1]
```diff
@@ -127,15 +127,24 @@ private void generateThumbnail(ImageParam imageParam) throws CommandValueExcepti
...
     }
 
+    /**
+     * Zips a folder of images.
+     * @param imageParam receives image params.
+     * @throws IOException when is a invalid file path.
+     */
     private void zipFile(ImageParam imageParam) throws IOException {
         PathJfc pathJfc = new PathJfc();
+        ZipFolder zip = new ZipFolder();
```
Comment (oscarlopez8) [inline src/main/java/com/jalasoft/jfc/model/image/ImageConverter.java:137]: Document this variable

File: src/main/java/com/jalasoft/jfc/model/image/ImageConverter.java [hunk 2]
```diff
@@ -127,15 +127,24 @@ private void generateThumbnail(ImageParam imageParam) throws CommandValueExcepti
...
 
-        File[] files = new File(imageParam.getOutputPathFile() + "/" + imageParam.getFolderName() +
-                "/").listFiles();
+        final String BACK_SLASH = "/";
```
Comment (oscarlopez8) [inline src/main/java/com/jalasoft/jfc/model/image/ImageConverter.java]: Back_Slash is one word, chante to backslash

File: src/main/java/com/jalasoft/jfc/model/image/ImageConverter.java (hunk 1 above)
Comment (Enrique-C) [reply src/main/java/com/jalasoft/jfc/model/image/ImageConverter.java:137]: It is not necessary in class methods

File: src/main/java/com/jalasoft/jfc/model/image/ImageConverter.java [hunk 3]
```diff
@@ -136,13 +136,13 @@ private void zipFile(ImageParam imageParam) throws IOException {
...
+        final String BACKSLASH = "/";
         final String ZIP_TAG = ".zip";
 
-        File[] files = new File(imageParam.getOutputPathFile() + BACK_SLASH + imageParam.getFolderName() +
-                BACK_SLASH).listFiles();
+        File[] files = new File(imageParam.getOutputPathFile() + BACKSLASH + imageParam.getFolderName() +
+                BACKSLASH).listFiles();
 
-        File fileZip = new File( pathJfc.getPublicFilePath() + BACK_SLASH + imageParam.getFolderName()+ ZIP_TAG);
+        File fileZip = new File( pathJfc.getPublicFilePath() + BACKSLASH + imageParam.getFolderName()+ ZIP_TAG);
```
Comment (oscarlopez8) [inline src/main/java/com/jalasoft/jfc/model/image/ImageConverter.java]: Add a space before "+ ZIP_TAG"

**Review by oscarlopez8 — APPROVED** [review]

**Pull request merged**